app/forms/declaration/DepartureTransport.scala

```diff
@@ -56,6 +65,18 @@ object TransportCodes {
   val InlandWaterway = "8"
   val Unknown = "9"
 
+  val extractModeOfTransportValue: String => String = _ match {
```

Comment:
I'd prefer if we made this an enum with a `code` and a `name`

Reply:
Removed